### PR TITLE
#43 state, model package 통합

### LINKED ISSUE #43: state, model 패키지 함치기
- model로 합친다.

### PR DESCRIPTION
**Pull request by JayYi**
## Description
- closed #43

### CODE REVIEW

File: presentation/src/main/java/com/dcs/presentation/core/model/UiState.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.dcs.presentation.core.state
```
Comment (JayYi) [inline presentation/src/main/java/com/dcs/presentation/core/model/UiState.kt]: 이 친구는 model이 아닌것 같은데....음...다시 생각해봐야겠네요

Comment (JayYi) [reply presentation/src/main/java/com/dcs/presentation/core/model/UiState.kt]: `ui.common`으로 옮길께요

Comment (JayYi) [reply presentation/src/main/java/com/dcs/presentation/core/model/UiState.kt]: 이동 완료
https://github.com/Edgar-J-Shin/AllAboutMovie/pull/47/commits/83f54ac1f7ed11c770b24143615a599979befaed

Comment (Edgar-J-Shin) [reply presentation/src/main/java/com/dcs/presentation/core/model/UiState.kt]: 이건 core > ui > state 로 빼면 되지 않을까?

Comment (JayYi) [reply presentation/src/main/java/com/dcs/presentation/core/model/UiState.kt]: `ui.state`로 변경했슴돠
https://github.com/Edgar-J-Shin/AllAboutMovie/pull/47/commits/c5c11f6ea7502027f4159c3b48a7d59d63bedb42

**Review by Edgar-J-Shin — APPROVED** [review]
Lgtm~

**Pull request merged**